packages/app/src/App.tsx

```diff
@@ -86,6 +86,8 @@ import {
   UserNotificationSettingsCard,
 } from '@backstage/plugin-notifications';
 
+import '@backstage/canon/css/styles.css';
+
```

Comment:
Bit of a weird one, but we also need to add this to `src/index-public-experimental.tsx` in both `app` and `app-next`.

Reply:
Done